_opensearch/rest-api/snapshots/restore-snapshot.md

```diff
@@ -0,0 +1,135 @@
+---
+layout: default
+title: Restore Snapshot
+parent: Snapshot APIs
+grand_parent: REST API reference
+nav_order: 9
+---
+
+## Restore Snapshot
+
+Restores a snapshot of a cluster or specified data streams and indices. 
+
+* For information about indices and clusters, see [Introduction to OpenSearch]({{site.url}}{{site.baseurl}}/opensearch/index).
+
+* For information about data streams, see [Data streams]({{site.url}}{{site.baseurl}}/opensearch/data-streams).
+
+If open indices with the same name that you want to restore already exist in the cluster, you must close, delete, or rename the indices. See [Sample Request](#sample-request-1) for information about renaming an index. See [Close index]({{site.url}}{{site.baseurl}}/opensearch/rest-api/index-apis/close-index) for information about closing an index.
+{: .note}
+
+### Path parameters
+
+| Parameter | Data Type | Description |
+:--- | :--- | :---
+repository | String | Repository containing the snapshot to restore. |
+| snapshot | String | Snapshot to restore. |
+
+### Query parameters
+
+Parameter | Data Type | Description
+:--- | :--- | :---
+wait_for_completion | Boolean |  Whether to wait for snapshot restoration to complete before continuing. |
+
+#### Sample request
+
+The following request restores `my-first-snapshot` located in `my-opensearch-repo`:
+
+````json
+POST /_snapshot/my-opensearch-repo/my-first-snapshot/_restore
+````
+
+#### Sample response
+
+Upon success, the response returns the following JSON object:
+
+````json
+{
+  "snapshot" : {
+    "snapshot" : "my-first-snapshot",
+    "indices" : [ ],
+    "shards" : {
+      "total" : 0,
+      "failed" : 0,
+      "successful" : 0
+    }
+  }
+}
+````
+
+### Request body
```

Comment:
Please rename the Request body section to Request fields and move it to right after the Query parameters section.

Reply:
I have made this change.